site-src/v1alpha2/guides/multiple-ns.md

```diff
@@ -63,90 +64,71 @@ administration to ensure that Routes are not over-exposed.
 
```

Comment:
> Route-owners can specify that they will bind with all Gateways in the cluster,
or only Gateways from a specific Namespace, with a specific label selector, or
an individual Gateway.

This is outdated. Route owners now target specific gateways only.

> Similarly, Gateways provide the same level of control.

Please expand on this. Gateways trust namespaces where Routes (which bind to it) can live.

Reply:
49cd40e46f2554bffc359c02b8565d8edfa0f109 was created to clean up the wording, and expand the wording as requested.

lmkwyt

Reply:
49cd40e46f2554bffc359c02b8565d8edfa0f109 was created to clean up the wording, and expand the wording as requested.